portfolio/src/main/webapp/script.js

```diff
@@ -12,22 +12,72 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
+// /**
+//  * Adds a random greeting to the page.
```

Comment:
It's generally discouraged to commit "dead" code - such as commented out code. If the code might be needed later, it's already in the version control system, so there's no need to keep it in the current version.

Reply:
Good to know